src/main/resources/fxml/rgb_renderer.fxml

```diff
@@ -15,42 +15,46 @@
   ~ the License.
   -->
 
-<?import javafx.scene.control.*?>
+<?import javafx.beans.property.SimpleBooleanProperty?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.control.Spinner?>
 <?import javafx.scene.layout.*?>
```

Comment:
Since you got rid of `<?import javafx.scene.control.*?>` want to git rid of `<?import javafx.scene.layout.*?>` and add the following.
```XML
<?import javafx.scene.layout.StackPane?>
<?import javafx.scene.layout.VBox?>
<?import javafx.scene.layout.HBox?>
<?import javafx.scene.layout.GridPane?>
```